modules/serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile_{context}"]
+= Scaling up workflows with the gitops profile
+
+Workflows with the `gitops` profile that were previously scaled down must be scaled back up to continue operation.
+
+.Procedure
+
+. Modify the `my-workflow.yaml` CR and scale up each workflow to `one` as shown in the following example:
++
+[source,yaml]
+----
+spec:
+  podTemplate:
+    replicas: 1
+----
+
+. Apply the updated CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <my-workflow.yaml> -n <target_namespace>
+----
+
+. (Optional) Scale the workflow back to `1` by running the following command:
++
+[source,terminal]
+----
+$ oc patch sonataflow <my-workflow> -n <target_namespace> --type=merge -p '{"spec": {"podTemplate": {"replicas": 1}}}'
```

Comment:
```suggestion
$ oc patch sonataflow <workflow_name> -n <target_namespace> --type=merge -p '{"spec": {"podTemplate": {"replicas": 1}}}'
```

If <my-workflow> is the workflow name to be entered by users, consider the placeholder